src/StorageSync/StorageSync/ServerEndpoint/NewServerEndpointCommand.cs

```diff
@@ -236,6 +236,7 @@ public override void ExecuteCmdlet()
                     VolumeFreeSpacePercent = VolumeFreeSpacePercent,
                     ServerLocalPath = ServerLocalPath,
                     ServerResourceId = ServerResourceId,
+                    TierFilesOlderThanDays = TierFilesOlderThanDays,
```

Comment:
Need a changelog entry describing this change, otherwise LGTM

Reply:
done